ui/components/component-library/tag/tag.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable jest/require-top-level-describe */
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { Tag } from './tag';
+
+describe('Tag', () => {
+  it('should render the label inside the tag', () => {
+    const { getByTestId } = render(<Tag data-testid="tag" label="Imported" />);
+    expect(getByTestId('tag')).toBeDefined();
+    expect(getByTestId('tag')).toHaveTextContent('Imported');
+  });
+});
```

Comment:
Probably worth while to do a snapshot test for our design system components, so that if any styles/classes etc change that it triggers a review/snapshot update

Reply:
Yeah, that makes sense. We can probably keep it for housekeeping epic @georgewrmarshall?

Reply:
Good idea @NidhiKJha! I'll add it to the house keeping issue and as a checklist item for each component https://github.com/MetaMask/metamask-extension/issues/15627